lib/app/widgets/scaffold.dart

```diff
@@ -42,6 +42,8 @@ class FancyScaffold extends StatelessWidget {
   }
 }
 
+typedef AppBarBuilder = Widget Function(bool innerBoxIsScrolled);
```

Comment:
```suggestion
typedef AppBarBuilder = Widget Function(bool isInnerBoxScrolled);
```
Maybe name the parameter according to Effective Dart Style Guides?